# double type x y yaw control duration
float64 x
float64 y
float64 yaw

float64 controlX
float64 controlY
float64 controlYAW

float64 duration

float64 pre_x
float64 pre_y
float64 pre_yaw

bool flag
bool tf_flag

bool diff_x
bool diff_y
bool diff_yaw
